test/components/selection-list.spec.tsx

```diff
@@ -309,58 +280,4 @@ describe('<SelectionList />', () => {
             });
         });
     });
-
-    describe(`Styling`, () => {
```

Comment:
These tests also have nothing to do with the theme. On the other hand, the following do:
* Renders blank items at the same height as normal items
* Renders items under each other using the default renderer

Reply:
can you make them pass without theme?

Reply:
Sure, if this can wait until Sunday, I can try.

Reply:
good!